test/JDBC/expected/BABEL-4390-vu-verify.out

```diff
@@ -82,3 +82,28 @@ GO
 
 ~~ERROR (Message: Cannot drop the procedure 'sp_testlinkedserver', because it does not exist or you do not have permission.)~~
 
+
+-- psql
+-- should fail
+DROP PROCEDURE dbo.xp_qv;
```

Comment:
are we adding hook because we want to block this use case? And shouldn't this be master_dbo?